client/components/QuestionsAndAnswers/ModalForm.jsx

```diff
@@ -1,51 +1,238 @@
-import React from 'react';
+import React, { useState } from 'react';
+import ReactDOM from 'react-dom';
 import styled from 'styled-components';
 
-// for add question
-const ModalForm = () => (
-  <form>
-    <Container1>
-      <LabelArea htmlFor="question">
-        Your Question
-      </LabelArea>
-      <LargeText
-        id="question"
-        type="text"
-        maxlength="1000"
-      />
-    </Container1>
-    <Container1>
-      <LabelArea htmlFor="nickname">
-        Your Nickname
-      </LabelArea>
-      <InputArea
-        id="nickname"
-        type="text"
-        maxlength="60"
-        placeholder="Example: jackson11!"
+export const QuestionModal = ({ setOpen, handleAddQuestion }) => {
+  // const [localData, setLocalData] = useState(question);
+  const [newQuestion, setNewQuestion] = useState({
+    question_body: '',
+    asker_name: '',
+    question_helpfulness: 66666,
+    email: '',
+    answers: {},
+  });
+
+  return (
+    ReactDOM.createPortal(
+      <>
+        <ModalShadow />
+        <Modal>
+          <h4>
+            Ask Your Question
+          </h4>
+          <h5>
+            About the [Procut Name Here]
+          </h5>
+          <QuestionForm
+            setNewQuestion={setNewQuestion}
+            newQuestion={newQuestion}
+            handleAddQuestion={handleAddQuestion}
+            setOpen={setOpen}
+          />
+        </Modal>
+      </>,
+      document.getElementById('app-modal'),
+    )
+  );
+};
+
+export const AnswerModal = ({ question, setIsAdd, handleAddAnswer }) => {
+  const [newAnswer, setNewAnswer] = useState({
+    body: '',
+    answerer_name: '',
+    date: '2018-01-04T00:00:00.000Z', // need refactor;
+    helpfulness: 333,
+    email: '',
+    photos: [],
+  });
+  return (
+    ReactDOM.createPortal(
```

Comment:
I'm not sure if we're allowed to use this for our modals - please check!

Reply:
Gonna double check on Monday, but Julie didn't say no during the meeting, haha,